buckets/local/bucket.go

```diff
@@ -89,10 +91,17 @@ func (b *Bucket) load() error {
 	}
 	if len(h.Roots) > 0 {
 		b.root = h.Roots[0]
+		b.cidver = int(b.root.Version())
 	}
 	return nil
 }
 
+// CidVersion returns the configured cid version (0 or 1).
+// The default is 1.
+func (b *Bucket) SetCidVersion(v int) {
+	b.cidver = v
+}
```

Comment:
So this is used to be able to match the local bucket with the cid version used remotely?